package.json

```diff
@@ -2,7 +2,8 @@
     "name": "@odyzeo/popup",
     "version": "3.1.0",
     "description": "Odyzeo popup component with Vue",
-    "main": "dist/popup.umd.js",
+    "main": "dist/popup.umd.min.js",
+    "unpkg": "dist/popup.umd.min.js",
```

Comment:
toto nemusi byt linka na https://unpkg.com/ ? ci ? co je to unpkg ? :D  

https://areknawo.com/whats-what-package-json-cheatsheet/

unpkg
Maybe it's not 100% official, but UNPKG as NPM-based CDN have gained so much popularity and user base, that this property might be worth a closer look. unpkg should point to a file that will later be exclusively used by UNPKG to provide its CDN support. It's usually the same as the earlier-mentioned browser.

Reply:
To je len alternativny service k npm. Inak je to taky isty subor ako v pripade `browser` atributu. V nasom pripade je to universal build. 

Inak som prebehol mnozstvo vselijakych for a tutorialov ci kniznic a v mnohych pripadoch sa vyberaju cestou rollupu a vsetkych moznych buildov. Nemyslim si ze je to nutne v nasom pripade. Ten UMD build totiz nema viac ako zopar KB (2.5 + 2.5 KB CSS).